pypit/artrace.py

```diff
@@ -2464,20 +2506,26 @@ def trace_slits(slf, mstrace, det, pcadesc="", maskBadRows=False, min_sqm=30.):
         rcen = rcent[ww, :].T.copy()
         extrapord = np.zeros(lcen.shape[1], dtype=np.bool)
 
-    # Remove any slits that are completely off the detector
```

Comment:
Hmmm... I now see how this is being interpreted by @jhennawi - is there are reason the other implementation wasn't working as expected?

Reply:
I think there must be.  Let's chat on this offline